### PR TITLE
0.12.2

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 0.12.2
- Build system is now scikit-build
- Dropped the entry_point as this is just a convenience for running tests and requires pytest anyway.

Version: 0.12.2. Released: Mar 2022
Release notes: https://github.com/KaveIO/PhiK/blob/master/CHANGES.rst
Repository: https://github.com/kaveio/phik
Documentation: https://phik.readthedocs.io

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,58 +1,60 @@
...
 
 source:
-  url: https://github.com/KaveIO/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: e98e6078b8c81165c6ad565b933883bda8370a375936c478f2d733bcd67863a8
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/phik-{{ version }}.tar.gz
+  sha256: b0674eb829d23290433f718d2360122be804b170ccc807ad9d9a8d1c13985533
 
 build:
-  number: 1
-  noarch: python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:14]: Skip `py<37`, https://github.com/KaveIO/PhiK/blob/932f5402a1e5cd0eec948193ea7679092e036243/setup.py#L101

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,58 +1,60 @@
...
-  entry_points:
-    - phik_trial = phik.entry_points:phik_trial  # [win]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:16]: `entry_points` are still relevant https://github.com/KaveIO/PhiK/blob/932f5402a1e5cd0eec948193ea7679092e036243/setup.py#L119-L126

**Review by skupr-anaconda — COMMENTED** [review]
This file is redundant https://github.com/AnacondaRecipes/phik-feedstock/blob/27110f2da5eb5aea77b2474c9cc0086c507c1729/recipe/example.py as we do not use it in the test environment

File: recipe/meta.yaml (hunk 2 above)
Comment (cbouss) [reply recipe/meta.yaml:16]: phik_trial is a convenience to call pytest and run the test suite of phik. 
But the tests are not included in the final package, and pytest is not a run dependency. 
So this is not usable anyway.

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:14]: Is this required since we support python 3.7 onwards?

Comment (skupr-anaconda) [reply recipe/meta.yaml:14]: It's only for clarification and sync with the upstream source

Comment (cbouss) [reply recipe/meta.yaml:14]: not convinced but I added it

**Review by skupr-anaconda — APPROVED** [review]

**Review by varlackc — COMMENTED** [review]
We need to make some adjustments in the `skip` section 
https://github.com/AnacondaRecipes/phik-feedstock/blob/368873d809a8d792f28ae6c24d3c8317fed1816e/recipe/meta.yaml#L15

We want to add a space between the `#` symbol and the `[py<37]` section

**Pull request merged**